### PR TITLE
[frontend] catch ActiveXML::Transport::Error

### LINKED ISSUE #3732: [production][package#save_meta] <status code="400">
  <summary>repoid 'openSUSE:13.2:Update' is illegal</summary>
</status>
package#save_meta threw this 2 times
<pre>
&lt;status code=&quot;400&quot;&gt;
  &lt;summary&gt;repoid &#39;openSUSE:13.2:Update&#39; is illegal&lt;/summary&gt;
&lt;/status&gt;

</pre>
[https://build.opensuse.org/package/save_meta/home:Miuku:VirtualBox/virtualbox](https://build.opensuse.org/package/save_meta/home:Miuku:VirtualBox/virtualbox)"

[See this exception on Errbit](https://errbit-opensuse.herokuapp.com/apps/5620ca2bdc71fa00b1000000/problems/59a5c69b81ce5100061cb389)

### PR DESCRIPTION
**Pull request by alexandergraul**
Fixes #3732.

### CODE REVIEW

File: src/api/app/controllers/webui/package_controller.rb [hunk 1]
```diff
@@ -1002,7 +1002,7 @@ def save_meta
         errors << "Package doesn't exists in that project."
       end
 
-    rescue Suse::ValidationError => e
+    rescue Suse::ValidationError, ActiveXML::Transport::Error => e
```
Comment (hennevogel) [inline src/api/app/controllers/webui/package_controller.rb]: What can produce an `ActiveXML::Transport::Error` here? `Suse::Validator.validate`?

Comment (hennevogel) [reply src/api/app/controllers/webui/package_controller.rb]: "Not sure" is not good enough, sorry...

**Comment by alexandergraul** [comment]
@hennevogel I found the method call that raises the exception (and learned to use errbit's backtrace)

**Comment by alexandergraul** [comment]
codecav diff coverage complaints do not matter, the rescue logic is tested somewhere else

**Comment by hennevogel** [comment]
@alexandergraul please rebase so we can merge this

**Pull request merged**